flight/Modules/OnScreenDisplay/onscreendisplay.c

```diff
@@ -1276,6 +1263,8 @@ void render_user_page(OnScreenDisplayPageSettingsData * page)
 		AccelsData accelsData;
 		AccelsGet(&accelsData);
 		// apply low pass filter to reduce noise bias
+		static AccelsData accelsDataAcc = {};
```

Comment:
Empty initializer lists are naughty right? http://stackoverflow.com/questions/17589533/is-an-empty-initializer-list-valid-c-code